### DISCUSSION TITLE
Azure for Students - K-12 students?

### DISCUSSION THREAD

**Discussion by mr-chris-reed** [microsoft/IoT-For-Beginners#425] [Q&A] [answered]
Hello,

Do you have any suggestions for accessing the Azure IoT resources for high school students?  I have a small group (3 students) and we are working through this your excellent IoT for Beginners curriculum.  MS Azure account age restriction is going to make it difficult to get them accounts.  We are getting close to the point where will want to start using Azure, if possible.  Any thoughts?

Thank you.

**Comment by mr-chris-reed** [comment]
@jimbobbennett - any thoughts on this?

**Comment by jimbobbennett** [comment, marked as answer]
@mr-chris-reed unfortunately, there isn't anything available to students under 18 that covers the IoT services. I've been requesting this repeatedly, but it's not yet happened.

My suggestion to you would be for you to set up the Azure subscription using the free tier, and do the Azure side yourself in collaboration with the students (so they see what you are doing, but you do the actual work and control the access). They can then connect their devices to your subscription.

I wish there was a better way, but we just don't have the Azure support for high school kids that we need.

**Reply by mr-chris-reed** [reply]
Okay, thanks @jimbobbennett , that sounds like a good solution.

**Comment by jimbobbennett** [comment]
Will also add - the offer for under 18s is the student starter - https://azure.microsoft.com/en-us/offers/ms-azr-0144p/

This does include Azure functions, just not iot hub
